test/mml_builder.js

```diff
@@ -656,7 +656,8 @@ suite('mml_builder use_workers=' + useWorkers, function() {
   });
 
   // See https://github.com/Vizzuality/grainstore/issues/62
-  test('throws useful error message on invalid text-name', function(done) {
+  // skiping since carto#0.16.3 does not throw error
+  test.skip('throws useful error message on invalid text-name', function(done) {
```

Comment:
OK, I see, thank you for the context. But then, we should either remove the test or make it run for the specific reference version that triggers that behavior, right?